common/OpTestIPMI.py

```diff
@@ -554,3 +555,74 @@ def ipmi_get_occ_status(self):
             raise OpTestError(l_msg)
 
         return l_result
+
+    ##
+    # @brief This function will deactivates ipmi sol console
+    #
+    # @return l_con it is a console object or raise OpTestError
```

Comment:
specify the type of return object. Will the return object be a string or some sort of an object